iib/workers/tasks/build.py

```diff
@@ -1498,7 +1498,13 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     """
     package_name, labels = _apply_package_name_suffix(metadata_path, organization)
 
-    operator_manifest = OperatorManifest.from_directory(manifests_path)
+    try:
+        operator_manifest = OperatorManifest.from_directory(manifests_path)
+    except ruamel.yaml.YAMLError:
+        error = 'metadata/annotations/yaml is not valid YAML'
+        log.exception(error)
+        raise IIBError(error)
```

Comment:
Instead of using generic error message, wouldn't be better to use concrete error message returned by  ruamel?

Like `f"not a valid yaml file: {exc}"`

Reply:
Changed